axon-framework/axon-framework-commands/infrastructure.md

```diff
@@ -414,59 +376,242 @@ public DisruptorCommandBus commandBus(TransactionManager txManager, AxonConfigur
+There are a couple of concepts that are configurable, regardless of the type of distribute command bus used:
 
-By default, [`SimpleCommandBus`](infrastructure.md) is configured to handle incoming commands on the different JVM's. You can configure `AxonServerCommandBus` to use other command bus implementations for this purposes: [`AsynchronousCommandBus`](infrastructure.md), [`DisruptorCommandBus`](infrastructure.md).
+### Local Segment
 
-### DistributedCommandBus
+Unlike the [local](#the-command-bus---local) `CommandBus` implementations, the distributed command buses do not invoke any handlers at all.
+All they do is form a "bridge" between command bus implementations on different JVM's, delegating any received commands to the so-called _local segment_.
 
-`DistributedCommandBus` is an alternative approach to distributing command bus \(commands\). Each instance of the `DistributedCommandBus` on each JVM is called a "Segment".
+By default, this local segment is the [`SimpleCommandBus`](infrastructure.md#simplecommandbus).
+You can configure the local segment to be any of the other local command buses too, like the [`AsynchronousCommandBus`](infrastructure.md#asynchronouscommandbus) and [`DisruptorCommandBus`](infrastructure.md#disruptorcommandbus).
+How to configure the local segment will be shown in the sections of the implementations
 
-![Structure of the Distributed Command Bus](../../.gitbook/assets/distributed-command-bus.png)
+### Load Factor
 
-The `DistributedCommandBus` relies on two components: a `CommandBusConnector`, which implements the communication protocol between the JVM's, and the `CommandRouter`, which chooses a destination for each incoming command. This router defines which segment of the `DistributedCommandBus` should be given a \`command, based on a routing key calculated by a routing strategy. Two commands with the same routing key will always be routed to the same segment, as long as there is no change in the number and configuration of the segments. Generally, the identifier of the targeted aggregate is used as a routing key.
+The load factor defines the amount of load an Axon application would carry compared to other instances.
+For example, if you have a two machine set up, both with a load factor of 100, both will carry an equal amount of load.
 
-Two implementations of the `RoutingStrategy` are provided: the `MetaDataRoutingStrategy`, which uses a metadata property in the command message to find the routing key, and the `AnnotationRoutingStrategy`, which uses the `@TargetAggregateIdentifier` annotation on the Command Messages payload to extract the routing key. Obviously, you can also provide your own implementation.
+Increasing the load factor to 200 on both would still mean that both machines receive the same amount of load.
+This points out that the load factor will only change load amongst systems if the values are in equal.
```

Comment:
```suggestion
This points out that the load factor will only change the load amongst systems if the values are in equal.
```
I am not entirely sure what this sentence means though.

Reply:
I feel like `are in` should be `aren't` ?

Reply:
Both your pointers are right.

What I am trying to share through that sentence, that the `loadFactor` only impacts a different load per running Axon Framework instance, if the values per instance actually differ.

If on both instances the loadFactor is a 100, both will receive the arbitrary value of a 100.
Doubling the value on both to 200 would thus not make a difference, as both would still receive 50% of the total load.